assets/app/lib/params.rb

```diff
@@ -9,6 +9,7 @@ def self.[](key)
     end
 
     def self.add(route, k, v)
+      route = route.split('#')[0]
```

Comment:
this looks similar to before

Reply:
This includes the commits from the other PR. I've fixed it there and rebased this one.

Reply:
I just realised that this strategy will probably fail because merging PRs changes the commits, so I'll probably have to rebase again when the other is merged.

If you want to merge them at the same time, you probably could just merge this one. (I'm sorry, I'm experienced with git but not with github and especially with PRs)